app/config/tstvhq/adventures.json

```diff
@@ -26,66 +26,68 @@
                 },
                 {
                     "id": "catacombs",
-                    "name": "Through the looking glass",
-                    "text": "You appear to be the only one who sees that you have inadvertently opened a secret door. You descend down the spiral staircase within. The door closes behind you. Descending, you find yourself in an ante-chamber with a single door.",
+                    "name": "a path of temptations",
+                    "text": "Donning the cloak you trepadaciously enter the secret door. Descending down a staircase you find yourself outside. A path leads into the woods. You find a note in the pocket of the cloak. It reads: ‘Whatever you do, don’t stray from the path. Love, Mom.’",
                     "buttons": [
                         {
-                            "text": "Open the door",
+                            "text": "Into the Woods!",
                             "nextRoom": ["fortune"]
                         },
                         {
-                            "text": "Look for a way out",
+                            "text": "Go back",
                             "nextRoom": ["library"]
                         }
                     ]
                 },
                 {
                     "id": "fortune",
-                    "name": "Open the door",
-                    "text": "The door will not open. It is locked. You can hear it though.",
+                    "name": "Encounter with the wolf",
+                    "text": "As you make your way through the trees you hear a gruff voice call to you from beyond the tree line along the path. ‘These woods have such beauty within them. Come pick some flowers.’ Delightful fragrant scents fill the air and you are tempted to see what kind of flowers could produce such a lovely bouquet.",
                     "buttons": [
                         {
-                            "text": "stop and listen",
+                            "text": "Pick flowers”,
```

Comment:
the quote here is bad (directional, maybe you can disable smartquotes in keyboard prefs?), it messes up the rest

Reply:
Fixed. So close. Hopefully we're working on stuff to make these kinds of things less of an issue this very morning.